src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1193,7 +1193,19 @@ public ResponseEntity<String> getMapEquipments(
             @Parameter(description = "Substations id") @RequestParam(name = "substationId", required = false) List<String> substationsIds,
             @Parameter(description = "Should get in upstream built node ?") @RequestParam(value = "inUpstreamBuiltParentNode", required = false, defaultValue = "true") boolean inUpstreamBuiltParentNode) {
 
-        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getMapEquipments(studyUuid, nodeUuid, substationsIds, inUpstreamBuiltParentNode));
+        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(studyService.getMapEquipments(studyUuid, nodeUuid, substationsIds, inUpstreamBuiltParentNode, "map-equipments"));
```

Comment:
Maybe rename "map equipement" into "voltage levels lines" everywhere ? (method name, path, etc... in all services)